server/pbench/bin/pbench-move-unpacked

```diff
@@ -47,7 +47,11 @@ echo $TS
 list=$UNPACK_PATH/pbench-unpack-tarballs.$$
 tmp=$TMP/$PROG.$$
 mail_content=$tmp/mail_content.log
-find $ARCHIVE/*/$linksrc -name '*.tar.xz' -printf "%s\t%p\n" 2>/dev/null | grep -v DUPLICATE | sed 's/\/'$linksrc'//' | sort -n > $list
+
+# find command will not generate any result if the linksrc is empty, and because of that du will not get any result 
+# to calculate the size. In that scenario du generates the size of the current directory as output, to handle
+# that exclude the current directory.
+find $ARCHIVE/*/$linksrc -type l -name '*.tar.xz' -printf "%l\n" 2>/dev/null | grep -v DUPLICATE | xargs du -b --exclude "." 2>/dev/null | sort -n > $list
 
 trap 'rm -rf $list $tmp' EXIT INT QUIT
```

Comment:
There's a problem here: the trap and mkdir of $tmp have to be moved before the find, since $list  is going to be created in $tmp after the conflict is resolved.